app/graphql/mutations/wallets/create.rb

```diff
@@ -23,10 +23,19 @@ def resolve(**args)
 
         result = ::Wallets::CreateService
           .new(context[:current_user])
-          .create(**args.merge(organization_id: current_organization.id))
+          .create(
+            **args
+              .merge(organization_id: current_organization.id)
+              .merge(customer: customer(args[:customer_id]))
+              .except(:customer_id)
```

Comment:
why we do an except here?

Reply:
since we only want to send customer object to the service and not the customer_id anymore (by default args contains customer_id)